### PR TITLE
docs(templates): Add documentation to functions in argtemplates #15

### PR DESCRIPTION
**Pull request by mrporsev**
Solves #15

### CODE REVIEW

File: httpie/cli/argtemplate.py [hunk 1]
```diff
@@ -95,6 +99,12 @@ def edit_json_template(args):
 
 
 def load_template(arg):
+    """
+    Load a template from templates.json and return a list of arguments to be passed to the main function
+    the format for running this is:
+        http <name> <item> <value>
```
Comment (editf) [inline httpie/cli/argtemplate.py]: I think this is wrong, to run a template you just do `http runt <template>`

Comment (mrporsev) [reply httpie/cli/argtemplate.py]: Thanks @editf, I'll fix it!

File: httpie/cli/argtemplate.py [hunk 2]
```diff
@@ -54,6 +54,10 @@ def store_json_template(args):
 
 
 def edit_json_template(args):
+    """
+    Edit a template in templates.json, the format for running this is:
+        http template <name> <item> <value>
```
Comment (editf) [inline httpie/cli/argtemplate.py]: oh I missed this before but this should be `http editt ...`

Comment (mrporsev) [reply httpie/cli/argtemplate.py]: nice catch, fixed it!

**Review by editf — APPROVED** [review]

**Pull request merged**